DetourTileCache/Include/DetourTileCache.h

```diff
@@ -130,8 +139,14 @@ class dtTileCache
 	
 	dtStatus removeTile(dtCompressedTileRef ref, unsigned char** data, int* dataSize);
 	
+	// Cylinder obstacle.
 	dtStatus addObstacle(const float* pos, const float radius, const float height, dtObstacleRef* result);
+
+	// Aabb obstacle.
 	dtStatus addBoxObstacle(const float* bmin, const float* bmax, dtObstacleRef* result);
+
+	// Obb obstacle: can be rotated in Y.
+	dtStatus addBoxObstacle(const float* bmin, const float* bmax, const float yRadians, dtObstacleRef* result);
```

Comment:
`bmin`/`bmax` here should be `center` and `halfExtents`. Though can we rename `halfExtents` to just `extents`? This is the convention used in other places (eg `findNearestPoly`). To make it more clear, here is an example: assuming the rotation is 0, passing center = `0, 0, 0` and `extents = 1, 1, 1` should mark the box from `-1, -1, -1` to `1, 1, 1`.